include/openmc/particle.h

```diff
@@ -69,6 +69,8 @@ struct NuclideMicroXS {
   double absorption;       //!< absorption (disappearance)
   double fission;          //!< fission
   double nu_fission;       //!< neutron production from fission
+  double prompt_nu_fission;//!< prompt neutron production from fission
+  std::vector<double> delayed_nu_fission;//!< delayed neutron production from fission
```

Comment:
Should probably use an `std::array<double, 8>` here since it's of a modest size and of known max size ahead of time. It will kill my GPU code performance to put an `std::vector` here. :smile: